client/src/components/ReviewsRatings/NewReview.jsx

```diff
@@ -1,45 +1,62 @@
 
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import Review from './review.jsx';
 import { Modal, Button, Form } from 'react-bootstrap';
 import Rating from '@material-ui/lab/Rating';
-const NewReview = ({ productId }) => {
+import $ from 'jquery';
+import axios from 'axios';
+import API_KEY from '/config.js';
 
+const NewReview = ({ reviews, productId, reviewChar }) => {
   const [show, setShow] = useState(false);
   const [starValue, setStarValue] = useState(0);
   const [email, setEmail] = useState('');
   const [name, setName] = useState('');
-  const [size, setSize] = useState(0);
-  const [width, setWidth] = useState(0);
-  const [comfort, setComfort] = useState(0);
-  const [quality, setQuality] = useState(0);
-  const [length, setLength] = useState(0);
-  const [fit, setFit] = useState(0);
+  const [reviewCharValues, setReviewCharValues] = useState({});
   const [recommend, setRecommend] = useState('');
   const [reviewSummary, setReviewSummary] = useState('');
   const [reviewBody, setReviewBody] = useState('');
   const [fileList, setFileList] = useState([]);
   const handleClose = () => setShow(false);
   const handleShow = () => setShow(true);
 
+  let results = {
```

Comment:
Real question, because I don't know: can we a const here instead of a let?

Reply:
Im not too sure. I changed it though. But my understanding of a constant variable is something that will not change or be set a new value.

Reply:
That's my understanding also, I just didn't know if this would only change given a new execution context.